saskatoon/member/filters.py

```diff
@@ -39,8 +39,7 @@ def lookups(self, request, model_admin):
 
     def queryset(self, request, queryset):
         if self.value():
-            group = Group.objects.get(name='volunteer')
-            return queryset.filter(groups__in=[group]).exclude(password__exact='')
+            return queryset.filter(id__in=[ user.id for user in queryset if user.is_onboarding ])
```

Comment:
let's not change this, introducing a for loop slows things down, it's always better to work with querysets as much as possible

Reply:
i feel like this might catch us if `is_onboarding` conditions ever change. this is only one place though, so it wouldn't be too hard to change if needed, just sitting in back of mind

Reply:
it's fine, the filter is only used for admins as a way to monitor what is happening.
Fee free to add a comment though with `is_onboarding` in there so when we grep in the future we catch it